### DISCUSSION TITLE
Limit amount of numbers/characters allowed for the input

### DISCUSSION THREAD

**Discussion by vaklinzi** [uNmAnNeR/imaskjs#567] [Q&A] [answered]
Hello, I would like to know how I can limit the amount of numbers for the input? For example, I have a currency field in which I would like to limit the user to 10 digits but I can't find how I can set it.

**Comment by uNmAnNeR** [comment, marked as answer]
hi! you can probably use `max`option if you are using number mask

**Comment by vaklinzi** [comment]
Yes, that helped me :) Thank you!
